interface-definitions/include/accel-ppp/gateway-address-multi.xml.i

```diff
@@ -0,0 +1,17 @@
+<!-- include start from accel-ppp/gateway-address-multi.xml.i -->
+<leafNode name="gateway-address">
+  <properties>
+    <help>Gateway IP address</help>
+    <constraintErrorMessage>invalid IPv4 address</constraintErrorMessage>
+    <valueHelp>
+      <format>ipv4net</format>
+      <description>Default Gateway, mask send to the client</description>
+    </valueHelp>
+    <constraint>
+      <validator name="ipv4-prefix"/>
```

Comment:
Do we really allow _network_ addresses like `192.0.2.0/25` as gateway addresses? Technically, in peer-to-peer networks, it may work, but I'd like to clarify that.

Reply:
Accel-PPP allows it. I have checked it on ipoe and it works.
```
set service ipoe-server authentication mode 'noauth'
set service ipoe-server client-ip-pool TEST range '10.0.0.3-10.0.0.7'
set service ipoe-server gateway-address '10.0.0.0/25'
```
On client side:
```
PC2> ip dhcp
DORA IP 10.0.0.3/25 GW 10.0.0.0

PC2> show ip

NAME        : PC2[1]
IP/MASK     : 10.0.0.3/25
GATEWAY     : 10.0.0.0
DNS         :
DHCP SERVER : 10.0.0.0
DHCP LEASE  : 595, 600/300/525
MAC         : 00:50:79:66:68:01
LPORT       : 20025
RHOST:PORT  : 127.0.0.1:20026
MTU         : 1500

PC2> ping 1.1.1.1

84 bytes from 1.1.1.1 icmp_seq=1 ttl=64 time=0.833 ms
84 bytes from 1.1.1.1 icmp_seq=2 ttl=64 time=1.421 ms
```
I think it is a network engineer's responsibility to set a proper IP address.